src/node/core_test.go

```diff
@@ -304,16 +304,16 @@ func TestSync(t *testing.T) {
 
 	/*
 
-	   core 0           core 1          core 2
-
-	                                    |   |  e20
-	                                    |   | / |
-	                                    |   /   |
-	                                    | / |   |
-	   e01 |   |                        e01 |   |
-	   | \ |   |                        | \ |   |
-	   e0  e1  |        |   e1  |       e0  e1  e2
-	   0   1   2        0   1   2       0   1   2
+		   core 0           core 1          core 2
```

Comment:
ASCII-art is broken :(

Reply:
Current Fantom-foundation/go-lachesis#master:
https://github.com/Fantom-foundation/go-lachesis/blob/cdbac6c/src/node/core_test.go#L307-L316

My branch for this PR:
https://github.com/SamuelMarks/go-lachesis/blob/53903214f50a63f64cad9e4f91f6720373443783/src/node/core_test.go#L307-L316